src/pages/QuestionPage.jsx

```diff
@@ -254,26 +47,36 @@ const QuestionPage = () => {
   };
 
   // Update the question status on answer click (either correct or incorrect)
-  const handleAnswerClickWithStatus = (isCorrect, questionIndex) => {
+  const handleAnswerClickWithStatus = (questionIndex) => {
```

Comment:
put this in context but look at my psedo code the function should be resposbile once clicked on an answer it will update the answers and update the notifcation.